### PR TITLE
Update lab-09-4-xor_tensorboard.py

### PR DESCRIPTION
**Pull request by qoocrab**
Simplify code and delete unnecessary variables.

### CODE REVIEW

File: lab-09-4-xor_tensorboard.py [hunk 1]
```diff
@@ -3,55 +3,45 @@
...
-Y = tf.placeholder(tf.float32, [None, 1], name='y-input')
-
-with tf.name_scope("layer1"):
-    W1 = tf.Variable(tf.random_normal([2, 2]), name='weight1')
-    b1 = tf.Variable(tf.random_normal([2]), name='bias1')
+
+x_data = np.array([[0, 0], [0, 1], [1, 0], [1, 1]], dtype=np.float32)
+y_data = np.array([[0], [1], [1], [0]], dtype=np.float32)
+
+X = tf.placeholder(tf.float32, [4, 2], name="x")
```
Comment (kkweon) [inline lab-09-4-xor_tensorboard.py]:
```suggestion
X = tf.placeholder(tf.float32, [None, 2], name="x")
```

File: lab-09-4-xor_tensorboard.py [hunk 2]
```diff
@@ -3,55 +3,45 @@
...
+Y = tf.placeholder(tf.float32, [4, 1], name="y")
```
Comment (kkweon) [inline lab-09-4-xor_tensorboard.py]:
```suggestion
Y = tf.placeholder(tf.float32, [None, 1], name="y")
```

File: lab-09-4-xor_tensorboard.py [hunk 3]
```diff
@@ -3,55 +3,45 @@
...
+with tf.name_scope("Layer1"):
+    W1 = tf.Variable(tf.random_normal([2, 2]), name="weight_1")
+    b1 = tf.Variable(tf.random_normal([2]), name="bias_1")
     layer1 = tf.sigmoid(tf.matmul(X, W1) + b1)
 
-    w1_hist = tf.summary.histogram("weights1", W1)
-    b1_hist = tf.summary.histogram("biases1", b1)
-    layer1_hist = tf.summary.histogram("layer1", layer1)
+    tf.summary.histogram("W1", W1)
+    tf.summary.histogram("B1", b1)
```
Comment (kkweon) [inline lab-09-4-xor_tensorboard.py]:
```suggestion
    tf.summary.histogram("b1", b1)
```

File: lab-09-4-xor_tensorboard.py [hunk 4]
```diff
@@ -3,55 +3,45 @@
...
+with tf.name_scope("Layer2"):
+    W2 = tf.Variable(tf.random_normal([2, 1]), name="weight_2")
+    b2 = tf.Variable(tf.random_normal([1]), name="bias_2")
     hypothesis = tf.sigmoid(tf.matmul(layer1, W2) + b2)
 
-    w2_hist = tf.summary.histogram("weights2", W2)
-    b2_hist = tf.summary.histogram("biases2", b2)
-    hypothesis_hist = tf.summary.histogram("hypothesis", hypothesis)
+    tf.summary.histogram("W2", W2)
+    tf.summary.histogram("B2", b2)
```
Comment (kkweon) [inline lab-09-4-xor_tensorboard.py]:
```suggestion
    tf.summary.histogram("b2", b2)
```

File: lab-09-4-xor_tensorboard.py [hunk 5]
```diff
@@ -64,20 +54,21 @@
...
-                  X: x_data, Y: y_data}), sess.run([W1, W2]))
+            print(step, cost_val)
 
     # Accuracy report
-    h, c, a = sess.run([hypothesis, predicted, accuracy],
-                       feed_dict={X: x_data, Y: y_data})
+    h, c, a = sess.run(
+        [hypothesis, predicted, accuracy], feed_dict={X: x_data, Y: y_data}
+    )
     print("\nHypothesis: ", h, "\nCorrect: ", c, "\nAccuracy: ", a)
```
Comment (kkweon) [inline lab-09-4-xor_tensorboard.py]:
```suggestion
     print(f"""
Hypothesis:
{h}
Predicted:
{p}
Accuracy: {a}""")
```

**Comment by qoocrab** [comment]
Thank you! I have revised it to reflect your suggestion.

**Pull request merged**